fineract-provider/src/main/java/org/apache/fineract/infrastructure/security/service/TwoFactorConfigurationServiceImpl.java

```diff
@@ -133,7 +133,7 @@ public boolean isSMSEnabled() {
     @Cacheable(value = "tfConfig", key = "T(org.apache.fineract.infrastructure.core.service.ThreadLocalContextUtil).getTenant().getTenantIdentifier()+'|smsProvider'")
     public Integer getSMSProviderId() {
         Integer value = getIntegerConfig(TwoFactorConfigurationConstants.SMS_PROVIDER_ID, null);
-        if (value < 1) {
+        if (value != null && value < 1) {
```

Comment:
This looks wrong to me. Should it be:
IF value is NULL OR value < 1 THEN return null?